src/index.ts

```diff
@@ -58,23 +50,23 @@ type CohereModule = {
   unknown[];
 
 const noop = () => {};
-const noopModule: CohereExports = {
-  init: noop,
-  identify: noop,
-  stop: noop,
-  showCode: noop,
-  getSessionUrl: noop,
-  makeCall: noop,
-  addCallStatusListener: noop,
-  removeCallStatusListener: noop,
-};
+const noopModule: CohereExports = Object.fromEntries(
+  bridgedMethods.map((method) => [[method], noop])
```

Comment:
Why is method [method]

Reply:
TypeScript complains that the key is a string instead of keyof CohereExports. But wrapping with [] doesn't change the output and passes TypeScript.